mikrotik/resource_dhcp_lease.go

```diff
@@ -32,6 +32,10 @@ func resourceLease() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"blocked": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Let's add `Default: "true"` to this also. I think the majority of cases will rely on the default value. This will also mean that not all of the test cases will need `blocked =` in them.

Reply:
Might be misunderstanding but by [`Default: "true"`,](terraform.io/docs/extend/schemas/schema-behaviors.html#default) aren't you suggesting we enable block-access by default?  Isn't the principal of least surprise: if optional values aren't specified, my DHCP lease will work in most cases (will not be blocked)?

Reply:
You're right I meant `Default: "false"`. What I want to avoid is requiring users to update their terraform code if all of their dhcp leases are not blocked.